enterprise/admin/__init__.py

```diff
@@ -47,11 +46,26 @@ class EnterpriseCustomerAdmin(DjangoObjectActions, SimpleHistoryAdmin):
     EXPORT_AS_CSV_FIELDS = ["name", "active", "site", "uuid", "identity_provider"]
```

Comment:
@haikuginger should probably add `catalog` field to `EXPORT_AS_CSV_FIELDS`.

bit: also, it might make sense to rename this const to `DISPLAY_FIELDS` and use it both for CSV export action and `list_display`